src/nwb_benchmarks/benchmarks/time_remote_slicing.py

```diff
@@ -58,6 +80,24 @@ def time_slice(self, s3_url: str, object_name: str, slice_range: Tuple[slice]):
         self._temp = self.data_to_slice[slice_range]
 
 
+class RemfileContinuousSliceBenchmarkWithCache:
+    rounds = 1
+    repeat = 3
+    param_names = param_names
+    params = params
+
+    def setup(self, s3_url: str, object_name: str, slice_range: Tuple[slice]):
+        self.nwbfile, self.io, self.file, self.bytestream, self.tmpdir = read_hdf5_nwbfile_remfile_with_cache(
+            s3_url=s3_url
+        )
+        self.neurodata_object = get_object_by_name(nwbfile=self.nwbfile, object_name="ElectricalSeriesAp")
```

Comment:
`object_name` should be set from input parameter instead of being hardcoded. It looks like this is a problem also in some of the existing tests. I'll submit a fix shortly.

Reply:
Fixed in [9147fd4](https://github.com/NeurodataWithoutBorders/nwb_benchmarks/pull/43/commits/9147fd47e682775033cbf9094e194d26e55f5761)